### ISSUE TITLE
Video thumbnails have artifacts

### ISSUE THREAD

**Issue by SavageCore** [SavageCore/new-retro-arcade-neon-attract-screen-tool#36]
`ffmpeg 3.2.4`

![ffmpeg 3.2.4](https://i.imgur.com/jFuOygj.jpg)

`ffmpeg 20180102-41e51fb` (bundled with `@ffmpeg-installer/ffmpeg 1.0.13`)

![ffmpeg 20180102-41e51fb](https://i.imgur.com/YHr33Sk.jpg)

Relevant `stderr` from `ffmpeg`

```
ffmpeg version N-89672-g41e51fbcd9 Copyright (c) 2000-2018 the FFmpeg developers
  built with gcc 7.2.0 (GCC)
  configuration: --enable-gpl --enable-version3 --enable-sdl2 --enable-bzlib --enable-fontconfig --enable-gnutls --enable-iconv --enable-libass --enable-libbluray --enable-libfreetype --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libopus --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libtheora --enable-libtwolame --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxml2 --enable-libzimg --enable-lzma --enable-zlib --enable-gmp --enable-libvidstab --enable-libvorbis --enable-libvo-amrwbenc --enable-libmysofa --enable-libspeex --enable-amf --enable-cuda --enable-cuvid --enable-d3d11va --enable-nvenc --enable-dxva2 --enable-avisynth --enable-libmfx
  libavutil      56.  7.100 / 56.  7.100
  libavcodec     58.  9.100 / 58.  9.100
  libavformat    58.  3.100 / 58.  3.100
  libavdevice    58.  0.100 / 58.  0.100
  libavfilter     7.  8.100 /  7.  8.100
  libswscale      5.  0.101 /  5.  0.101
  libswresample   3.  0.101 /  3.  0.101
  libpostproc    55.  0.100 / 55.  0.100
Input #0, mov,mp4,m4a,3gp,3g2,mj2, from 'C:\Users\SavageCore\contra.mp4':
  Metadata:
    major_brand     : isom
    minor_version   : 512
    compatible_brands: isomiso2avc1mp41
    title           : Contra (US / Asia, set 1) [MAME]
    artist          : Progetto VideoSnaps by motoschifo & AntoPISA [gameplay by AntoPISA]
    encoder         : Lavf54.20.4
    comment         : Created with: Mame version 0.163, 
                    : Played by: AntoPISA, 
                    : Game name: contra - Contra (US / Asia, set 1), 
                    : Date game played: 15 luglio 2015, 
                    : Date video creation: 4 novembre 2015, 
                    : Type: shortplay single-screen, 
                    : Manufacturer: Konami, 
                    : Display resolution: 224x280px (rotated by 90°), 
                    : Link for info: http://adb.arcadeitalia.net/?mame=contra, 
                    : Copyright (C) - All rights reserved - Tutti i marchi e loghi appartengono ai rispettivi proprietari
    genre           : MAME video preview (shortplay)
  Duration: 00:00:38.59, start: 0.000000, bitrate: 352 kb/s
    Stream #0:0(und): Video: h264 (High 4:4:4 Predictive) (avc1 / 0x31637661), yuv444p, 224x280 [SAR 15:16 DAR 3:4], 160 kb/s, 29.97 fps, 29.97 tbr, 2997 tbn, 59.94 tbc (default)
    Metadata:
      handler_name    : VideoHandler
    Stream #0:1(und): Audio: aac (LC) (mp4a / 0x6134706D), 48000 Hz, stereo, fltp, 185 kb/s (default)
    Metadata:
      handler_name    : SoundHandler
Stream mapping:
  Stream #0:0 -> #0:0 (h264 (native) -> mjpeg (native))
Press [q] to stop, [?] for help
[h264 @ 0000022d9e0546c0] top block unavailable for requested intra mode
[h264 @ 0000022d9e0546c0] error while decoding MB 10 0, bytestream 20543
[h264 @ 0000022d9e0546c0] concealing 252 DC, 252 AC, 252 MV errors in I frame
[h264 @ 0000022d9e054b40] cabac decode of qscale diff failed at 13 5
[h264 @ 0000022d9e054b40] error while decoding MB 13 5, bytestream 832
[h264 @ 0000022d9e054b40] concealing 218 DC, 218 AC, 218 MV errors in P frame
[h264 @ 0000022d9e0546c0] error while decoding MB 12 13, bytestream -15
[h264 @ 0000022d9e0546c0] concealing 107 DC, 107 AC, 107 MV errors in B frame
[swscaler @ 0000022d9e7940c0] deprecated pixel format used, make sure you did set range correctly
Output #0, image2, to 'C:\Users\SavageCore\AppData\Roaming\new-retro-arcade-neon-attract-screen-tool\thumbnails\contra.jpg':
  Metadata:
    major_brand     : isom
    minor_version   : 512
    compatible_brands: isomiso2avc1mp41
    title           : Contra (US / Asia, set 1) [MAME]
    artist          : Progetto VideoSnaps by motoschifo & AntoPISA [gameplay by AntoPISA]
    genre           : MAME video preview (shortplay)
    comment         : Created with: Mame version 0.163, 
                    : Played by: AntoPISA, 
                    : Game name: contra - Contra (US / Asia, set 1), 
                    : Date game played: 15 luglio 2015, 
                    : Date video creation: 4 novembre 2015, 
                    : Type: shortplay single-screen, 
                    : Manufacturer: Konami, 
                    : Display resolution: 224x280px (rotated by 90°), 
                    : Link for info: http://adb.arcadeitalia.net/?mame=contra, 
                    : Copyright (C) - All rights reserved - Tutti i marchi e loghi appartengono ai rispettivi proprietari
    encoder         : Lavf58.3.100
    Stream #0:0(und): Video: mjpeg, yuvj444p(pc), 224x280 [SAR 15:16 DAR 3:4], q=2-31, 200 kb/s, 29.97 fps, 29.97 tbn, 29.97 tbc (default)
    Metadata:
      handler_name    : VideoHandler
      encoder         : Lavc58.9.100 mjpeg
    Side data:
      cpb: bitrate max/min/avg: 0/0/200000 buffer size: 0 vbv_delay: -1
[h264 @ 0000022d9dfc5780] Reference 5 >= 2
[h264 @ 0000022d9dfc5780] error while decoding MB 3 17, bytestream 19
[h264 @ 0000022d9dfc5780] concealing 60 DC, 60 AC, 60 MV errors in B frame
[h264 @ 0000022d9e05ea40] Reference 6 >= 6
[h264 @ 0000022d9e05ea40] error while decoding MB 7 9, bytestream 967
[h264 @ 0000022d9e05ea40] concealing 168 DC, 168 AC, 168 MV errors in P frame
[h264 @ 0000022d9dfc5780] Reference 12 >= 10
[h264 @ 0000022d9dfc5780] error while decoding MB 4 15, bytestream 76
[h264 @ 0000022d9dfc5780] concealing 87 DC, 87 AC, 87 MV errors in P frame
[h264 @ 0000022d9e054b40] gray chroma
[h264 @ 0000022d9e054b40] error while decoding MB 6 17, bytestream 99
[h264 @ 0000022d9e054b40] concealing 57 DC, 57 AC, 57 MV errors in P frame
[h264 @ 0000022d9e0546c0] gray chroma
[h264 @ 0000022d9e0546c0] error while decoding MB 7 4, bytestream 340
[h264 @ 0000022d9e0546c0] concealing 238 DC, 238 AC, 238 MV errors in B frame
frame=    1 fps=0.0 q=2.0 Lsize=N/A time=00:00:00.03 bitrate=N/A speed=1.28x    
video:13kB audio:0kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: unknown
```

**Comment by SavageCore** [comment]
This is related to files encoded with SEI (https://trac.ffmpeg.org/ticket/6717#comment:1)

To fix a file we can re-encode like so `ffmpeg -i contra-bad.mp4 -c:v libx264 -profile high444 -pix_fmt yuv444p -an contra-fixed.mp4`

**Comment by SavageCore** [comment]
Closed via d7d294361a58f97bef7a0c556752eeac15316c7f
